nef-plugin/Utils/Result+Extensions.swift

```diff
@@ -0,0 +1,14 @@
+//  Copyright © 2020 The nef Authors.
+
+import Foundation
+
+extension Result {
+    
+    @discardableResult
+    public func fold<B>(_ fe: (Failure) -> B, _ fa: (Success) -> B) -> B {
+        switch self {
+        case let .failure(error): return fe(error)
+        case let .success(value): return fa(value)
+        }
+    }
+}
```

Comment:
This extension already exists in Bow 0.8.0

Reply:
ah! great